app/controllers/foreman_ansible/concerns/ansible_roles_controller_extensions.rb

```diff
@@ -38,11 +38,22 @@ def get_roles_variables(imported_variables, kind, role)
         variables_to_s(variables)
       end
 
+      def excluded_roles
+        @excluded_roles ||= Setting.convert_array_to_regexp(
+          Setting[:ansible_roles_to_ignore],
+          {
+            :prefix => '(\A|.*::|.*_)',
```

Comment:
Why `::` and `_`? I can't figure that out...

Reply:
that's not necessary. I removed it, thanks.